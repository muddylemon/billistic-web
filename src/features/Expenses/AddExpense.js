import React from "react"
import "./expense_list.scss"
import { compose } from "recompose"

import { withExpenseHandlers, withExpenseData } from './Expense.hocs'

import Modal from '@material-ui/core/Modal'
import Button from '@material-ui/core/Button'
// import Checkbox from '@material-ui/core/Checkbox'


export const AddExpense = ({ open }) => (
  <div>
    <Button>Open Modal</Button>
    <Modal
      aria-labelledby="simple-modal-title"
      aria-describedby="simple-modal-description"
    >
      <div>
        testing 123
      </div>
    </Modal>
  </div>
)

export default compose(
  withExpenseData,
  withExpenseHandlers
)(AddExpense)
