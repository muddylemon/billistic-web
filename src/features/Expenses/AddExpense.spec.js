import React from "react"
import { AddExpense } from "./AddExpense"
import ShallowRenderer from "react-test-renderer/shallow"

describe('AddExpense', () => {
  it('should render fine', () => {
    const tree = new ShallowRenderer().render(<AddExpense />)
    expect(tree).toMatchSnapshot()
  })
})
