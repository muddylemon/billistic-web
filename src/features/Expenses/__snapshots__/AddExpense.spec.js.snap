// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddExpense should render fine 1`] = `
<div>
  <WithStyles(Button)>
    Open Modal
  </WithStyles(Button)>
  <WithStyles(Modal)
    aria-describedby="simple-modal-description"
    aria-labelledby="simple-modal-title"
  >
    <div>
      testing 123
    </div>
  </WithStyles(Modal)>
</div>
`;
